src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = getReadFromFile(fromFileName);
+        String report = getReportToFile(dataFromFile);
+        getWriteToFile(report,toFileName);
+    }
+
+    private String getReadFromFile(String fromFileName) {
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder builder = new StringBuilder();
```

Comment:
FYI if you want you can also get all the lines to a list of strings like this:
bufferedReader.lines().collect(Collectors.toList());